org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/preferences/GradleEditorPreferencePage.java

```diff
@@ -0,0 +1,68 @@
+package org.eclipse.buildship.ui.preferences;
+
+import org.eclipse.buildship.ui.PluginImage.ImageState;
+import org.eclipse.buildship.ui.PluginImages;
+import org.eclipse.buildship.ui.UiPlugin;
+import org.eclipse.jface.preference.BooleanFieldEditor;
+import org.eclipse.jface.preference.ColorFieldEditor;
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+
+/**
+ * @author Benjamin gurok
+ * @author Paul Verest
+ */
+public class GradleEditorPreferencePage extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
```

Comment:
We have a simple editor and thus we should keep it simple. How about we
- remove the preference page
- reuse the colors  from the Java editor (`org.eclipse.jdt.ui.PreferenceConstants`)
- Add a preference listener to update the colors when the Java editor colors changed.
This way we can provide a consistent color scheme without adding more complexity to the UI.

Reply:
I hesitated reusing Java Colors. I will see if that fits.